Sprint fixes and changes

EebleSheembl opened the pull request:
Put new and working fatigue controller in HumanDefaultRagdoll.xml (Gives Stamina Affliction)
Added Stamina affliction, used to control fatigue amounts given based on speed
Changed Fatigue to be slower to lose the more progressed it is

EebleSheembl committed:
Update misc.xml

-Added sprint affliction to control fatigue
-Changed fatigue to make it slower to recover if you have alot

Signed-off-by: Eeble Sheembl <152429881+EebleSheembl@users.noreply.github.com>

EebleSheembl committed:
Update HumanDefaultRagdoll.xml

Got rid broken fatigue controller, added working one

Signed-off-by: Eeble Sheembl <152429881+EebleSheembl@users.noreply.github.com>

ven1883 commented on Afflictions/misc.xml:
```diff
@@ -288,7 +288,8 @@
 	
-	<!--++++++++++++++++++++++++++++++++++++-->
+	<!--++++++++++++++++++++++++++++++++++++
+ -->
```
The comment, eeble!!!!

ven1883 commented on Afflictions/misc.xml:
```diff
@@ -300,7 +301,7 @@
 		showicontoothersthreshold="1000"
 		showinhealthscannerthreshold="3"
-		showiconthreshold="3" 
+		showiconthreshold="3"
```
What
how

It's the same goddam thing?

EebleSheembl replied:
idk

EebleSheembl committed:
idk how that happened

fixed divider thingy

Signed-off-by: Eeble Sheembl <152429881+EebleSheembl@users.noreply.github.com>

ven1883 commented:
Functional but barely.
Because it relies on the movespeed of a character, the act of being fatigued prevents you from being fatigued further, which prevents it from capping out.

Also! This does not account for armor.
MOST armor sets slow you down, which makes this problematic.

EebleSheembl committed:
mfw idk what to put here

Made it slower to lose fatigue when moving
Made it that you get fatigue at gt 1.7

Signed-off-by: Eeble Sheembl <152429881+EebleSheembl@users.noreply.github.com>

ven1883 commented:
holgy shifted it worked....
